src/microbe_stage/MicrobeStage.tscn

```diff
@@ -2043,6 +2105,9 @@ visible = false
 [connection signal="mouse_exited" from="MicrobeHUD/BottomRight/EditorButton" to="MicrobeHUD" method="OnEditorButtonMouseExit"]
 [connection signal="pressed" from="MicrobeHUD/BottomRight/EditorButton" to="MicrobeHUD" method="EditorButtonPressed"]
 [connection signal="OnClosed" from="MicrobeHUD/ProcessPanel" to="MicrobeHUD" method="OnProcessPanelClosed"]
+[connection signal="pressed" from="MicrobeHUD/ScrollContainer/HotBar/Engulfment" to="PlayerMicrobeInput" method="ToggleEngulf"]
```

Comment:
Could you maybe add a comment on the PlayerMicrobeInput class that other callbacks directly call methods in it? Someone might otherwise get caught offguard when changing the methods around in there as this place would be a bit difficult to find.

Reply:
Added.